lib/smartsheet/rows.ex

```diff
@@ -11,4 +11,13 @@ defmodule Smartsheet.Rows do
 
     Smartsheet.ParseResponse.parse(__MODULE__, __ENV__.function, response)
   end
+
+  def update(sheet_id, rows) do
+    path = "/sheets/#{sheet_id}/rows"
+
+    {:ok, response = %HTTPoison.Response{}} =
+      Smartsheet.client().put(path, rows, "Content-Type": "application/json")
```

Comment:
is there a chance this can return :error?

Reply:
nope. internally, Smartsheet.client() is just a wrapper around HTTPoison, which always returns
```elixir
{:ok, %HTTPoison.Response{}}
```

even when the response wasn't successful.

however, when we parse the response, our client will respond with
```elixir
{:ok, %Smartsheet.Response{}} # for success
{:error, %Smartsheet.Response{}} # for errors
```